nkms_eth/actors.py

```diff
@@ -0,0 +1,263 @@
+from collections import OrderedDict
+from typing import Tuple, List
+
+# from nkms_eth.agents import MinerAgent, PolicyAgent
+from nkms_eth.base import Actor
+
+
+class PolicyArrangement:
+    def __init__(self, author: 'PolicyAuthor', miner: 'Miner', value: int,
+                 periods: int, arrangement_id: bytes=None):
+
+        if arrangement_id is None:
+            self.id = self.__class__._generate_arrangement_id()  # TODO: Generate policy ID
```

Comment:
So we'll generate a policy ID *and* an arrangement ID?  I don't have my head wrapped around that yet.

Reply:
Yes. I'm thinking that "arrangement ID" can be generated, in part, from `Policy` hrac. This may be important in order for Alice to revoke or track individual Ursulas in a single `Policy`.  There is some naming and composition to be done here... For example, the KMS names a relationship between Alice and one Ursula as `Contract`, which I propose is actually "BlockchainArrangement", a subclass of kms-eth `actors.BlockchainPolicyArrangement`.

Reply:
Yes, agreed, `nkms.policy.models.Contract` is actually `Arrangement`.